### PR TITLE
Skeleton init

### LINKED ISSUE #1: Initial project skeleton
use the template from MapColonies/work-standard

### PR DESCRIPTION
**Pull request by CptSchnitz**
closes #1

### CODE REVIEW

File: .gitignore copy [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Logs
```
Comment (syncush) [inline .gitignore copy:1]: what is the file purpose?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+COPY ./package*.json ./
+
+RUN npm install
+COPY . .
+RUN npm run build
+
+FROM node:12.18.3-slim as production
+
+ARG SERVICE_NAME=meow
```
Comment (syncush) [inline Dockerfile:14]: change name

File: package.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    "husky": "^4.2.5",
+    "jest": "^26.6.3",
+    "jest-create-mock-instance": "^1.1.0",
+    "jest-html-reporters": "^2.0.3",
+    "nodemon": "^2.0.6",
+    "rimraf": "^3.0.2",
+    "standard-version": "^9.0.0",
+    "supertest": "^6.0.1",
+    "ts-jest": "^26.3.0",
+    "typescript": "^3.8.3"
```
Comment (syncush) [inline package.json:73]: we should add prettier and remove nodemon for now until we find the proper configuration

File: src/common/middlewares/ErrorHandler.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  public getErrorHandlerMiddleware(): ErrorRequestHandler {
+    return (
+      err: Error,
+      req: Request,
+      res: Response,
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      next: NextFunction
+    ): void => {
+      this.logger.log('error', `${req.method} request to ${req.originalUrl}  has failed with error: ${err.message}`);
+      res.sendStatus(StatusCodes.INTERNAL_SERVER_ERROR);
```
Comment (syncush) [inline src/common/middlewares/ErrorHandler.ts:20]: according to the spec we are sending a body with a message property, we should follow the spec

**Review by syncush — APPROVED** [review]

File: .gitignore copy (hunk 1 above)
Comment (CptSchnitz) [reply .gitignore copy]: i copied the work-standard into the existing folder and i already had a gitignore :(
fixed

File: Dockerfile (hunk 2 above)
Comment (CptSchnitz) [reply Dockerfile]: fixed

File: package.json (hunk 3 above)
Comment (CptSchnitz) [reply package.json]: fixed

File: src/common/middlewares/ErrorHandler.ts (hunk 4 above)
Comment (CptSchnitz) [reply src/common/middlewares/ErrorHandler.ts]: fixed

**Pull request merged**